src/lib/fake.ts

```diff
@@ -6,43 +6,40 @@ import {
   seedDatabase,
   type ZustandDatabase,
 } from "lib/database/index.ts"
-import { type Server, startServer } from "lib/server.ts"
+
+import { createMakeRequest } from "./make-request.ts"
 
 export const createFake = async (): Promise<Fake> => {
   const database = createDatabase()
   return new Fake(database)
 }
 
 export class Fake {
-  public server: Server | null
-
-  public database: Database
+  public database: Omit<
+    Database,
+    "getState" | "setState" | "destroy" | "subscribe"
+  >
 
   readonly #database: ZustandDatabase
 
+  private readonly makeRequestWithDatabase: ReturnType<typeof createMakeRequest>
+
   constructor(database: ZustandDatabase) {
-    this.server = null
     this.#database = database
     this.database = database
-  }
-
-  async startServer({ port }: { port?: number } = {}): Promise<Server> {
-    this.server = await startServer({ port, database: this.#database })
-    return this.server
-  }
-
-  async stopServer(): Promise<void> {
-    this.server?.close()
-  }
-
-  get serverUrl(): string | null | undefined {
-    return this.server?.serverUrl
+    this.makeRequestWithDatabase = createMakeRequest(database)
   }
 
   async seed(): Promise<Seed> {
     return seedDatabase(this.#database)
   }
 
+  async makeRequest(request: Request): Promise<Response> {
```

Comment:
Where is Response defined? I don't see it imported anywhere.

Reply:
it's from @types/node, same as Request